plugin/pkg/scheduler/algorithmprovider/affinity/affinity.go

```diff
@@ -37,9 +37,9 @@ func affinityPredicates() util.StringSet {
 		"PodFitsResources",
 		"NoDiskConflict",
 		// Ensures that all pods within the same service are hosted on minions within the same region as defined by the "region" label
-		factory.RegisterFitPredicate("ServiceAffinity", algorithm.NewServiceAffinityPredicate(factory.PodLister, factory.ServiceLister, factory.MinionLister, []string{"region"})),
+		factory.RegisterFitPredicate("RegionAffinity", algorithm.NewServiceAffinityPredicate(factory.PodLister, factory.ServiceLister, factory.MinionLister, []string{"region"})),
```

Comment:
@bgrant0607 I would only like to clarify the statements around not running kubernetes across zone.  I believe the statement is that you should run kubernetes across small enough area that you can afford to lose (at least the master tier for some time period) and that has low enough network latency to communicate master <-> node and node <-> node for your applications.  For some deployments that might mean cluster can't cross zone.  For others, cross zone would be desired for higher availability of applications without having to maintain multiple kubernetes clusters.  And the more highly available kubernetes masters are, the more likely cross zone would be popular for smaller deployments.